src/main/java/dev/morling/kccli/service/Context.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright 2021 The original authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package dev.morling.kccli.service;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.net.URI;
+
+public class Context {
+    private final URI cluster;
+    private final String bootstrapServers;
```

Comment:
Should it be a list of strings (not sure from the top of my head about what the Kafka client expects)?

Reply:
Consumer and Producer Configs expect this as a single, comma-separated string 😢

Reply:
Ok, gotcha. Let's leave it as is then.